.storybook/preview.js

```diff
@@ -13,13 +13,13 @@ const preview = {
       options: {
         // Default colors
         light: { name: 'Light', value: '#FFFFFF' },
-        dark: { name: 'Dark', value: '#333' },
+        dark: { name: 'Dark', value: '#1d1d1b' },
         // Custom colors
-        'midnight-blue': { name: 'Midnight Blue', value: '#005380'},
```

Comment:
You'll find references to `Midnight Blue`, `Charcoal`, etc in some stories. They are used to set the stories background and will need to be replaced 🙂